code/modules/mob/living/basic/vermin/crab.dm

```diff
@@ -88,8 +88,8 @@
 	idle_behavior = /datum/idle_behavior/idle_random_walk
 	planning_subtrees = list(
 		/datum/ai_planning_subtree/find_nearest_thing_which_attacked_me_to_flee/from_flee_key,
-		/datum/ai_planning_subtree/flee_target/from_flee_key,
 		/datum/ai_planning_subtree/target_retaliate/to_flee,
```

Comment:
Actually on review maybe this subtree is just redundant with the one above it and it always worked because there's no point having both anyway?

Reply:
`find_nearest_thing_which_attacked_me_to_flee/from_flee_key` was bugged and didn't set the right target key
`target_retaliate/to_flee` would have worked but was placed after the flee behaviour
by fixing `find_nearest_thing_which_attacked_me_to_flee/from_flee_key` (as I did in this PR) I can remove the redundant extra behaviour which was also unused